$OpenBSD: patch-test,v 1.1 2008/01/06 17:09:36 merdely Exp $

This patch works around a strange problem with macppc.  Even though the
regression tests pass, there is an output problem that causes
"make regress" to fail.

--- test.orig	Sun Jan  6 11:47:52 2008
+++ test	Sun Jan  6 11:48:19 2008
@@ -137,6 +137,8 @@ def test_with_unittest():
     unittest_summary = Summary()
     doctest_summary = Summary()
 
+    sys.stdout = open('/dev/null')
+
     if unittests:
         print "Running unittests..."
         for relpath in unittests:
